src/tests/naming/test_altcoin.cairo

```diff
@@ -0,0 +1,429 @@
+use array::ArrayTrait;
+use array::SpanTrait;
+use option::OptionTrait;
+use zeroable::Zeroable;
+use traits::{Into, TryInto};
+use starknet::testing;
+use starknet::ContractAddress;
+use starknet::contract_address::ContractAddressZeroable;
+use starknet::contract_address_const;
+use starknet::testing::{set_contract_address, set_block_timestamp};
+use identity::{
+    identity::main::Identity, interface::identity::{IIdentityDispatcher, IIdentityDispatcherTrait}
+};
+use openzeppelin::token::erc20::{
+    interface::{IERC20Camel, IERC20CamelDispatcher, IERC20CamelDispatcherTrait}
+};
+use naming::interface::naming::{INamingDispatcher, INamingDispatcherTrait};
+use naming::interface::pricing::{IPricingDispatcher, IPricingDispatcherTrait};
+use naming::naming::main::Naming;
+use naming::pricing::Pricing;
+use naming::naming::utils::UtilsImpl;
+use super::common::{deploy, deploy_stark};
+use super::super::utils;
+use core::debug::PrintTrait;
+use wadray::Wad;
+
+#[test]
+#[available_gas(200000000000)]
+fn test_convert_quote_to_eth() {
+    let mut unsafe_state = Naming::unsafe_new_contract_state();
+
+    // User wants to buy a domain in STRK for one year
+    let domain_price_eth = Wad { val: 8999999999999875 };
+    // 1 STRK = 0,00522180500429277 ETH
```

Comment:
Makes sense with the current `get_altcoin_price` but should be updated so altcoin is the quoted token in base currency: ETH